policy/lib/tekton/trusted.rego

```diff
@@ -101,11 +101,26 @@ _task_expires_on(task) := expires if {
 	expires = time.parse_rfc3339_ns(record.expires_on)
 }
 
+_unexpired_records(records) := all_unexpired if {
```

Comment:
Perhaps it's worth adding a small simple test to exercise/demonstrate _unexpired_records behavior, since that's a new rule being added here.